flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -687,61 +687,23 @@ class JobManager(
     message match {
 
       case RequestAccumulatorResults(jobID) =>
-        try {
```

Comment:
I have another question. I don't know much about Akka (hardly anything, really), and I was wondering, if you forward the message, would this try/catch (in the `JobManager` actor) be able to capture exceptions that were thrown in the archive actor? If not, then a try/catch should also be used in `MemoryArchivist`.